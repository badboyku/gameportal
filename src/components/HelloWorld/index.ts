/* istanbul ignore file */
export { default as HelloWorld } from './HelloWorld';
