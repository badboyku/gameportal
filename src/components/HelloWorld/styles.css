.hello-world {
  color: #7d61fb;
}
