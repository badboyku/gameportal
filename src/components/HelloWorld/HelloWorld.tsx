import './styles.css';

type Props = {};

const HelloWorld = (_props: Props) => {
  return <div className="hello-world">Hello World!</div>;
};

export default HelloWorld;
