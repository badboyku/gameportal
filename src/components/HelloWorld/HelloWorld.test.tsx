import { render, screen } from '@testing-library/react';
import HelloWorld from './HelloWorld';

const renderHelloWorld = (props = {}) => {
  return render(<HelloWorld {...props} />);
};

describe('Component HelloWorld', () => {
  it('renders without crashing', () => {
    const { asFragment } = renderHelloWorld();

    expect(asFragment()).toMatchSnapshot();
  });

  it('renders div with "Hello World!"', () => {
    renderHelloWorld();

    expect(screen.getByText('Hello World!')).toBeInTheDocument();
  });
});
