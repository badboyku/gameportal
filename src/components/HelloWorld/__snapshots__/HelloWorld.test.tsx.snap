// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component HelloWorld renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="hello-world"
  >
    Hello World!
  </div>
</DocumentFragment>
`;
